src/Be.Vlaanderen.Basisregisters.Api/StartupDefaults-Configure.cs

```diff
@@ -241,6 +241,8 @@ public class DefaultResources { }
 
                 .AddXmlDataContractSerializerFormatters()
 
+                .AddFormatterMappings()
```

Comment:
Instead of https://github.com/aspnet/Mvc/blob/1c4b0fcdf38320b2f02c0bb7c31df5bd391ace07/src/Microsoft.AspNetCore.Mvc.Core/DependencyInjection/MvcCoreMvcCoreBuilderExtensions.cs#L44 can you call https://github.com/aspnet/Mvc/blob/1c4b0fcdf38320b2f02c0bb7c31df5bd391ace07/src/Microsoft.AspNetCore.Mvc.Core/DependencyInjection/MvcCoreMvcCoreBuilderExtensions.cs#L50 and provide a hook? Similar to https://github.com/Informatievlaanderen/api/blob/f25f428d064335331236a2ee659aae588d80aa87/src/Be.Vlaanderen.Basisregisters.Api/StartupDefaults-Configure.cs#L236